### PR TITLE
ADT_Assignment_34_Done

### PR DESCRIPTION
**Pull request by AditiGJUN20**
### Assignment: 34
- Please review and Comment.
Screen Shot 2020-07-27 at 12 28 13 PM
Screen Shot 2020-07-27 at 12 27 55 PM

### CODE REVIEW

File: src/aditiGJUN20/ElementRemovalAndReplaceInArrayList.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+		return wordCount;
+	}
+
+	public static void main(String[] args) {
+		ElementRemovalAndReplaceInArrayList elementRemovalAndReplaceInArrayList = new ElementRemovalAndReplaceInArrayList();
+		String[] inputArray = { "Maulik", "Techno", "Krishna", "Techno", "Credits", "Techno" };
+		ArrayList<String> inputArrayList = new ArrayList<String>();
+		for (int index = 0; index < inputArray.length; index++) {
+			inputArrayList.add(inputArray[index]);
+		}
```
Comment (AmitaRJun20) [inline src/aditiGJUN20/ElementRemovalAndReplaceInArrayList.java:50]: Suggestion : 
We also can add elements in Arraylist like :-
ArrayList<String> inputArrayList= new ArrayList<String>(Arrays.asList("Maulik","Techno" ,"Krishna", "Techno", "Credits", "Techno"));

**Review by AmitaRJun20 — APPROVED** [review]
Code looks good..

File: src/aditiGJUN20/ElementRemovalAndReplaceInArrayList.java (hunk 1 above)
Comment (AditiGJUN20) [reply src/aditiGJUN20/ElementRemovalAndReplaceInArrayList.java:50]: Thank you for the suggestion @AmitaRJun20.
I'll try to incorporate this logic in future code.

**Review by KrishnaKTechnocredits — APPROVED** [review]
Good job dear.

**Pull request merged**